### PR TITLE
Update BarModel.java

### LINKED ISSUE #27: Define setters/getters for all fields in BarModel.java

### PR DESCRIPTION
**Pull request by Rohittt728**
Added getters and setters to BarModel.java

### CODE REVIEW

**Comment by Rohittt728** [comment]
Contributing to issue #27

File: library/src/main/java/in/krharsh17/barview/BarModel.java [hunk 1]
```diff
@@ -4,6 +4,38 @@
...
+
+    public void setColor (String color) {
+    	this.color = color;
+    }
+
+     public Float getFillRatio () {
+    	return this.fillRatio;
+    }
+
+    public void setFillRatio (Float fillRatio) {
```
Comment (krharsh17) [inline library/src/main/java/in/krharsh17/barview/BarModel.java]:
```suggestion
    public String getLabel() {
    	return this.label;
    }

    public void setLabel(String label) {
    	this.label = label;
    }

     public String getValue() {
    	return this.value;
    }

    public void setValue(String value) {
    	this.value = value;
    }

     public String getColor() {
    	return this.color;
    }

    public void setColor(String color) {
    	this.color = color;
    }

     public Float getFillRatio() {
    	return this.fillRatio;
    }

    public void setFillRatio(Float fillRatio) {
```

**Review by krharsh17 — CHANGES_REQUESTED** [review]

**Comment by krharsh17** [comment]
Also, you haven't linked the issue to this PR. Make sure to go through every guideline that has been sent on the slack channel for the project. Such behaviour will not be entertained again.

**Comment by Rohittt728** [comment]
Sorry for the same. This is my first open source contribution. Will take care next time.

**Comment by krharsh17** [comment]
Fixes #27 : This is how you link an issue to a PR

**Review by krharsh17 — APPROVED** [review]
Perfect now.

**Comment by Rohittt728** [comment]
Thanks for the learning and your response!

**Pull request merged**